appveyor.yml

```diff
@@ -3,7 +3,7 @@ version: 1.0.{build}
 branches:
   # whitelist
   only:
-    - master
+    - sam-cli-release
```

Comment:
Are we going to keep this new branch here, or this was used just for testing?

Reply:
This branch will be the one we are making changes to during release. We will merge it to master at the end of release.